niri-ipc/src/lib.rs

```diff
@@ -1224,6 +1226,13 @@ pub enum Event {
         /// workspaces are missing from here, then they were deleted.
         workspaces: Vec<Workspace>,
     },
+    /// A window on a workspace became urgent.
```

Comment:
I think this is better as just "The workspace urgency changed." because it could change due to window closing or moving away, etc.